lls/main.c

```diff
@@ -537,24 +526,18 @@ lls_stage2(void *arg)
 			return ret;
 	}
 
-	/* TODO Have a different block set up RSS on the back interface. */
 	if (lls_conf->nd_cache.iface_enabled(net_conf, &net_conf->back)) {
-		uint8_t port_in = net_conf->back.id;
-		uint16_t lls_queue = lls_conf->rx_queue_back;
-
-		ret = gatekeeper_setup_rss(port_in, &lls_queue, 1);
-		if (ret < 0)
-			return ret;
-
-		ret = gatekeeper_get_rss_config(port_in, &lls_conf->rss_conf);
-		if (ret < 0)
-			return ret;
-
 		ret = register_nd_acl_rules(&net_conf->back);
 		if (ret < 0)
 			return ret;
 	}
 
+	/*
+	 * The GK block sets up RSS on the back interface,
+	 * and classify + distribute ND packets to the LLS block using
+	 * the DPDK libraries.
+	 */
```

Comment:
Please drop this comment. Blocks that register for IPv6 filters shouldn't need to know who is sendint them the filtered IPv6 packets.